### ISSUE TITLE
Profiles Accessibility

### ISSUE THREAD

**Issue by amiranickerson** [utdal/profiles#47]
### To get to WCAG A

- [x] Page language has not been identified
- The main language of the page needs to declared within the <html> element 

- [x] Links without a text alternative
- Pictures need alt tags (Profile pictures, article images, etc.)

- Emails do not have aria labels
<img width="227" alt="Screen Shot 2022-05-10 at 3 40 57 PM" src="https://user-images.githubusercontent.com/69992600/167718068-93848b5a-d53b-4a49-93d0-41c14e2408ee.png">

- Example: [https://profiles.utdallas.edu/bokyung.park](https://profiles.utdallas.edu/bokyung.park)
<img width="1466" alt="Screen Shot 2022-05-10 at 3 46 59 PM" src="https://user-images.githubusercontent.com/69992600/167719008-c547cf71-3ccf-41e2-9828-29ef7a366da7.png">

- This page contains one or more "empty" headings (heading tags with no text content)
- Example: [https://profiles.utdallas.edu/zanjum](https://profiles.utdallas.edu/zanjum)
<img width="488" alt="Screen Shot 2022-05-10 at 3 50 21 PM" src="https://user-images.githubusercontent.com/69992600/167719546-152cc845-39e9-460d-bea4-1462bf4bf692.png">

### To get to AA

- [x] Color contrast is not sufficient
- example: [https://profiles.utdallas.edu/thomas.linehan](https://profiles.utdallas.edu/thomas.linehan)
<img width="442" alt="Screen Shot 2022-05-10 at 3 54 21 PM" src="https://user-images.githubusercontent.com/69992600/167720165-7f139f5c-5291-4f02-b170-16dca37015dd.png">
<img width="984" alt="Screen Shot 2022-05-10 at 3 56 18 PM" src="https://user-images.githubusercontent.com/69992600/167720415-c0af70b6-d6db-45d4-9af7-6a4e32b1eef1.png">
<img width="312" alt="Screen Shot 2022-05-10 at 3 58 00 PM" src="https://user-images.githubusercontent.com/69992600/167720647-df9c4c4d-a288-483d-86c8-baedfe12aa16.png">
<img width="437" alt="image" src="https://user-images.githubusercontent.com/69992600/167720754-87e189c7-2a43-42f2-aed5-5ac5de5e56dc.png">

**Comment by betsyecastro** [comment]
- [ ] Meet with Amira to go over text alternative attributes that were added and changes in background color to fix contrast issues.
- [ ] Add field for text alternative attribute to articles/news images.
- [ ] Validate existing content in the section before creating the html element to avoid empty headings or empty container elements.
- [ ] Add missing fieldset to major in student form.

**Comment by betsyecastro** [comment]
Profile show:
- [ ] User picture text alternative
- [ ] Url text alternative

Profile News Section:
- [ ] Update text alternative for current images?

Profile edit:
- [ ] Select Image button not activated with tab
- [ ] Description text area without text alternative

**Comment by betsyecastro** [comment]
@amiranickerson the most recent accessibility fixes have been deployed to the staging env https://ordev.utdallas.edu/profiles/ the description of the changes applied are in the PR #65 

Anything else that you find you can add it to this issue and I will organize it later maybe per category/status. 

Thank you.

**Comment by betsyecastro** [comment]
TODO
- [ ] Images without a text alternative
- [ ] Empty headings
- [ ] Input needs id="major"
<img width="468" alt="image" src="https://user-images.githubusercontent.com/69992600/167720735-5f3108e3-0f77-448c-b54d-6c05813e5faa.png">
- [ ] Missing fieldset
<img width="468" alt="image" src="https://user-images.githubusercontent.com/69992600/167720822-83d15e31-b56c-4f99-9806-36d740e9d463.png">

**Comment by wunc** [comment]
Closing out this issue. I created a new Issue #87 to handle all of the newly-identified accessibility issues.
